### PR TITLE
Add log redaction to learner toml

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add log redaction to learner toml

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/7438
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@shichun-0415 @hehechen 是 tiflash.toml 和 tiflash-learner.toml 里都有 redact_info_log 吗？

**Comment by shichun-0415** [comment]
> @shichun-0415 @hehechen 是 tiflash.toml 和 tiflash-learner.toml 里都有 redact_info_log 吗？

image

**Review by hehechen — APPROVED** [review]

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -197,6 +199,12 @@ delta_index_cache_size = 0
     snap-handle-pool-size = 2
     ## 控制 raft store 持久化 WAL 的最小间隔。通过适当增大延迟以减少 IOPS 占用，默认为 "4ms"，设为 "0ms" 则关闭该优化。
     store-batch-retry-recv-timeout = "4ms"
+
+[security]
+    ## 是否开启日志脱敏
+    ## 若开启该选项，日志中的用户数据会以 `?` 代替显示
+    ## 默认值为 false
+    redact_info_log = false
```
Comment (JaySon-Huang) [inline tiflash/tiflash-configuration.md:207]:
```suggestion
    redact-info-log = false
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**